helm-charts/templates/data-plane/gateway-components/gateway-runtime/default-jwt-issuer.yaml

```diff
@@ -53,6 +53,13 @@ spec:
     - remoteClaim: "organization"
       localClaim:  "x-wso2-organization"
     {{ end }}
+    { { if .Values.wso2.apk.idp.groupsClaim } }
```

Comment:
helm install failure might be due to space between '{ {'

Reply:
fixed with https://github.com/wso2/apk/pull/1826/commits/7efabfac443df2e71ad4042da8bffcbd28631ccd